modules/gapi/src/streaming/engine/base_engine.cpp

```diff
@@ -0,0 +1,93 @@
+#include "streaming/engine/base_engine.hpp"
+#include "streaming/vpl/vpl_accel_policy.hpp"
+#include "logger.hpp"
+
```

Comment:
Also I'm not sure about naming of those files/folders. I'd suggest to bind it to `vpl` somehow

Reply:
+1